libr/core/cbin.c

```diff
@@ -2205,7 +2205,7 @@ static int bin_symbols(RCore *r, int mode, ut64 laddr, int va, ut64 at, const ch
 			}
 			if (sn.demname) {
 				r_meta_set (r->anal, R_META_TYPE_COMMENT,
-							addr, symbol->size, sn.demname);
+							addr, symbol->size? symbol->size: 1, sn.demname);
```

Comment:
imho is better to have a separate variable instead of making that line that long.

```
const size_t sym_size = symbol->size > 0? symbol->size: 1;
```